tests/Utils/Models/Episode.php

```diff
@@ -0,0 +1,23 @@
+<?php declare(strict_types=1);
+
+namespace Tests\Utils\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+
+/**
+ * Primary key.
+ *
+ * @property int $id
+ *
```

Comment:
Missing `@property` tags for the other fields

Reply:
`/**
 * Primary key.
 *
 * @property int $id
 *
 * @property string $title
 *
 *  Timestamps
 * @property \Illuminate\Support\Carbon $schedule_at
**/`